src/main/java/io/github/cottonmc/cotton/gui/widget/WScrollBar.java

```diff
@@ -238,6 +238,14 @@ public WScrollBar setWindow(int window) {
 		return this;
 	}
 
+	public int getScrollingSpeed() {
+		return scrollingSpeed;
+	}
+
+	public void setScrollingSpeed(int scrollingSpeed) {
```

Comment:
`setScrollingSpeed` needs to check that the scrolling speed is positive and throw an IllegalArgumentException otherwise, negatives and zero aren't really good values for it